tests/console/postgresql_server.pm

```diff
@@ -14,13 +14,13 @@ use base "consoletest";
 use strict;
 use testapi;
 use utils;
-use version_utils 'sle_version_at_least';
+use version_utils qw/is_sle is_leap/;
 use apachetest;
 
 sub run {
     select_console 'root-console';
 
-    my $pgsql_server = sle_version_at_least('15') ? 'postgresql10-server' : 'postgresql96-server';
+    my $pgsql_server = (is_sle('>=12-sp4') or is_leap('15.0+')) ? 'postgresql10-server' : 'postgresql96-server';
```

Comment:
Now you are leaving Tumbleweed behind again. I would also recommend same as coolo to install the virtual package postgresql-server

Reply:
it's not virtual - it requires the default

Reply:
`postgresql-server` package is on the SLE15+  real package which requires `postgresql-server-implementation`

on sle12 this package doesn't exist and is capability provided by postgresqlNM-server packages